core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -227,7 +227,7 @@ public Iterable<AclRequests> findAclRequestsByExample(
     if (requestor != null && !requestor.isEmpty()) {
       request.setRequestor(requestor);
     }
-    if (requestOperationType != null) {
+    if (requestOperationType != null && !requestOperationType.value.equalsIgnoreCase("all")) {
```

Comment:
Nitpick but "all" should probably be repalced by requestOperationType != RequestOperationType.ALL

Reply:
@aindriu-aiven updated.